nemo_skills/training/nemo_rl/configs/grpo.yaml

```
@@ -110,9 +110,14 @@ policy:
 
 data:
   max_input_seq_length: ${policy.max_total_sequence_length} # upper bound, real truncation occurs at vllm.max_model_len
-  prompt_file: "/nemo_run/code/nemo_skills/training/nemo_rl/prompts/math.txt"
-  system_prompt_file: null
-  dataset_name: "OpenMathInstruct-2"
+  prompt:
+    prompt_config: 'qwen/math-cot'
```

let's not set defaults and instead ask users to specify always. Otherwise too easy to make a mistake and not notice

```suggestion
    prompt_config: ???
```